handlers/main.yml

```diff
@@ -1,3 +1,8 @@
 ---
 - name: update-initramfs
   command: 'update-initramfs -u'
+
+- name: restart-auditd
+  command:
+    cmd: 'service auditd restart' # rhel: see: https://access.redhat.com/solutions/2664811
+    warn: no # sadly 'service' module fails in that case also by using 'use: service'
```

Comment:
I just sucessfully tested this using the service-module on Debian 10 and Centos 7:
```
- name: restart-auditd
  service:
    name: auditd
    state: restarted
    use: service
```

Where did this not work for you?

Reply:
In my previous tests id did not work with service module... because I set an invalid "max_logfile_action" in audits.conf. Meh.
After your hint I retested and you're right, that works indeed. Was quite wondering.
changing back.

Reply:
hmm travis moans again :/

from Cents-7 test:

fatal: [localhost]: FAILED! => {"changed": false, "msg": "Failed to stop auditd.service: Operation refused, unit auditd.service may be requested by dependency only (it is configured to refuse manual start/stop).\nSee system logs and 'systemctl status auditd.service' for details.\nJob for auditd.service failed because the control process exited with error code. See \"systemctl status auditd.service\" and \"journalctl -xe\" for details.\n"}

this is exactly the error I thought I've also seen on my previous tests. Looks like the "use: service" parameter is ignored...
though my local tests worked so far :/ will try with the provided kitchen again.

Reply:
Well that's because of Docker + systemd not working properly together. I don't know how to solve that, yet.

Reply:
Ah, I see. well, then feel free to approve :=P

Reply:
If you put back the `command` in the service-module, I can approve.